napari/layers/labels/_labels_key_bindings.py

```diff
@@ -16,51 +19,79 @@ def hold_to_pan_zoom(layer):
         layer.mode = prev_mode
 
 
-@Labels.bind_key('P')
+def register_label_action(description, shortcuts):
```

Comment:
Can we turn these into a single `register_layer_action(layer_type, description, shortcuts)`. That seems like it will be much more reusable across the code base (could go in `layer_utils.py` file)

Reply:
I can, though I'll expect the end API to change a bit with the action manager handling the keybinding.

Reply:
Yeah, i think the current approach is ok. You could also make all these private `_register_label_action` for example, I don't have super strong feelings though, curious if others do